en/Live preview update.md

```diff
@@ -1,61 +1,59 @@
-Live Preview is introduced to help you preview what you're writing on the go, as well as hide Markdown formatting.
+Obsidian 13 introduces a brand new editor engine that comes with several enhancements to your editing experience:
 
-See Live Preview in action:
+- Live Preview
+- Spellchecker
```

Comment:
spellchecker used to work but only with English (US), now you can select languages and it will also work on multiple langs

not sure what's the best way to phrase that

Reply:
Hm. Maybe we should focus on Live Preview and cut out the other enhancements (spellchecker, text subst, and Vim mode) in this post? Unless you think they're necessary to explain LP.

Reply:
I don't think they're THAT essential, so let's focus on live preview on this page!